flow-info/src/flow_info.rs

```diff
@@ -103,11 +103,16 @@ impl From<FlowStatus> for AtomicFlowStatus {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub struct FlowInfoLocked {
-    // We need this to use downcast because VpcDiscriminant is in net.
-    // We could avoid this indirection by moving Packet into its own crate.
+    // We need this to use downcast because objects used in this function (such as VpcDiscriminant)
+    // are in other crates (such as "net"). We could avoid this indirection by moving Packet into
```

Comment:
If we are going to change this comment I would change it to say that we need this to avoid circular dependencies period.  We can move `Packet` into its own crate, and then avoid the conflict for VpcDiscriminant, but ultimately, the `nat` crate will depend on `Packet` and `Packet` will depend on stuff in the `nat` crate unless we have this indirection for most things.